cmd/publishing-bot/publisher.go

```diff
@@ -114,11 +114,16 @@ func (p *PublisherMunger) ensureCloned(dst string, dstURL string) error {
 		return nil
 	}
 
-	err := exec.Command("mkdir", "-p", dst).Run()
-	if err != nil {
+	cmd := exec.Command("mkdir", "-p", dst)
+	if err := p.plog.Run(cmd); err != nil {
+		return err
+	}
+	cmd = exec.Command("git", "clone", dstURL, dst)
+	if err := p.plog.Run(cmd); err != nil {
 		return err
 	}
-	cmd := exec.Command("git", "clone", "--no-tags", dstURL, dst)
```

Comment:
was the intent to avoid pulling down content we don't need? can we upgrade the version of git in the image instead?

Reply:
> can we upgrade the version of git in the image instead?

I would rather avoid pulling in any non-standard package. Our use of git is probably fragile. We are on a Debian LTS release for a reason.

> was the intent to avoid pulling down content we don't need?

We don't want `v1.11.0` like tags. Instead we fetch them "manually" in a controlled way and rewrite them to `kubernetes-1.11.0`.